pkgs/swift2objc/lib/src/transformer/transformers/transform_property.dart

```diff
@@ -85,24 +81,16 @@ List<String> _generateSetterStatemenets(
   final wrappedInstanceProperty =
       '${wrappedClassInstance.name}.${originalProperty.name}';
 
-  if (originalProperty.type.isObjCRepresentable) {
-    assert(originalProperty.type.id == transformedProperty.type.id);
+  if (originalProperty.type.id == transformedProperty.type.id) {
```

Comment:
Can this check go in `generateUnwrappedValue` too?

Reply:
Done